draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -228,14 +232,14 @@ beginning of ICE gathering;
 - the addition of remote ICE host candidates with mDNS names generates mDNS
 queries for names of each candidate;
 
-- the removal of names could happen when the execution context of the ICE agent
+- the removal of names could happen when the browsing context of the ICE agent
 is destroyed in an implementation, and goodbye responses should be sent to
```

Comment:
Shouldn't should be SHOULD? E.g. make it a recommendation to invalidate mDNS records?

Reply:
That's probably to strong considering we allow mDNS names to be pre-generated/allocation.